etc/avocado/avocado.conf

```diff
@@ -83,3 +83,10 @@ skip_broken_plugin_notification = []
 # options or test types).
 # The keyword "@DEFAULT" will be replaced with all available unused loaders.
 loaders = ['file', '@DEFAULT']
+
+# Options that will only have effect when the
+# corresponding optional plugin is installed
+[optional.plugins]
+# Plugin: resultsdb
+# URL of the resultsdb API
+# resultsdb_api = http://resultsdb.example.com/api/v2.0/
```

Comment:
Ideally, there will be no references to "optional" things inside the main (core) avocado configuration file.  That's why the drop-in (avocado.conf.d) exist.  It'll certainly make the deployment of the optional plugin a lit bit more complex, but I think it's the way to go.  It also matches the approach used with vt, vt-joblock, etc.